uwreact_msgs/msg/RobotEntity.msg

```diff
@@ -0,0 +1,10 @@
+uint8 ALLIANCE_UNKNOWN = 0
+uint8 ALLIANCE_OPPOSING = 1
+uint8 ALLIANCE_PARTNER = 2
+
+Header        header
+GenericEntity entity
+uint8         alliance
+uint8         team_number
+float32       has_cargo_confidenceasdfasdfa
```

Comment:
Wtf is this name

Reply:
Smh it was part of an experiment I was doing and I forgot to remove it. Thanks for pointing it out, fixed.